deploy/crds/toolchain_v1alpha1_nstemplatetier_crd.yaml

```diff
@@ -10,6 +10,8 @@ spec:
   names:
     kind: NSTemplateTier
     plural: nstemplatetiers
+    shortNames:
```

Comment:
should we also have `singular: nstemplatetier` to be consistent with the plural name?

Reply:
isn't it set by default - that you can use lowercase `nstemplatetier` without specifying it in `shortNames`?

Reply:
Yes. `nstemplatetier` works OOTB. No need to add it to `shortNames` explicitly.

Reply:
ok, sorry for the dumb question 🤦‍♂